DataFormats/Detectors/FIT/FT0/src/DataFormatsFT0LinkDef.h

```diff
@@ -28,4 +28,8 @@
 #pragma link C++ class o2::ft0::HitType + ;
 #pragma link C++ class vector < o2::ft0::HitType> + ;
 
+#pragma link C++ class o2::ft0::RawEventData + ;
+#pragma link C++ class o2::ft0::EventHeader + ;
+#pragma link C++ class o2::ft0::EventData + ;
+
```

Comment:
if the lookup table should be persistent, you should also add std::vector<o2::ft0::Topo> to relevant LinkDef